content/en/serverless/azure_app_services.md

```diff
@@ -130,25 +130,107 @@ DogStatsd.Increment("sample.startup");
+
+### Installing the extension for the first time {#powershell-first-time}
+
+The install script adds the latest version of the extension to an Azure Web App or Azure Function App. This occurs on a per-instance basis, rather than at a resource group level.
+
+1. Open the Azure CLI or Azure Cloud Shell.
+2. Download the installation script using the following command:
+
+    ```
+    Invoke-WebRequest -Uri "https://raw.githubusercontent.com/DataDog/datadog-aas-extension/master/management-scripts/extension/install-latest-extension.ps1" -OutFile "install-latest-extension.ps1"
+    ```
+
+3. Run the following command, passing in required arguments and any others as needed.
+
+    ```
+    .\install-latest-extension.ps1 -Username <USERNAME> -Password <PASSWORD> -SubscriptionId <SUBSCRIPTION_ID> -ResourceGroup <RESOURCE_GROUP_NAME> -SiteName <SITE_NAME> -DDApiKey <DATADOG_API_KEY> -DDSite <DATADOG_SITE> -DDEnv <DATADOG_ENV> -DDService <DATADOG_SERVICE> -DDVersion <DATADOG_VERSION>
+    ```
+
+**Note**: The following arguments are required for the above command:
+
+- `<USERNAME>`: Your Azure user scope username.
+- `<PASSWORD>`: Your Azure user scope password.
+- `<SUBSCRIPTION_ID>`: Your Azure [subscription ID][22].
+- `<RESOURCE_GROUP_NAME>`: Your Azure resource group name.
+- `<SITE_NAME>`: The name of your app.
+- `<DATADOG_API_KEY>`: Your [Datadog API key][12].
+
+Also, set `DATADOG_SITE` to your [Datadog site][23]. `DATADOG_SITE` defaults to `datadoghq.com`. Your site is: {{< region-param key="dd_site" code="true" >}}.
+
+### Updating the extension for a resource group {#powershell-resource-group}
+
+The update script applies to an entire resource group. This script updates every instance (Web App or Function App) that has the extension installed. App Service instances that do not have the Datadog extension installed are not affected.
+
+1. Open the Azure CLI or Azure Cloud Shell.
+2. Download the installation script using the following command:
+
+    ```
+    $baseUri="https://raw.githubusercontent.com/DataDog/datadog-aas-extension/master/management-scripts/extension"; Invoke-WebRequest -Uri "$baseUri/update-all-site-extensions.ps1" -OutFile "update-all-site-extensions.ps1"; Invoke-WebRequest -Uri "$baseUri/install-latest-extension.ps1" -OutFile "install-latest-extension.ps1"
+    ```
+
+3. Run the following command. All arguments are required.
+
+    ```
+    .\update-all-site-extensions.ps1 -SubscriptionId <SUBSCRIPTION_ID> -ResourceGroup <RESOURCE_GROUP_NAME> -Username <USERNAME> -Password <PASSWORD>
+    ```
+
+### Installing a specific version of the extension {#powershell-specific-version}
+
+The Azure App Service UI does not support the ability to install a specific version of an extension. You may do this with the install or update script.
+
+#### Install specific version on a single resource
```

Comment:
Headers are using `Installing` and `Install`. I would recommend choosing one to make the page consistent (same thing with `Update` vs `Updating`).